WS2012R2/lisa/remote-scripts/ica/Core_LIS_CD.sh

```diff
@@ -139,12 +117,27 @@ if [ 0 -ne ${sts} ]; then
     LogMsg "Unable to unmount the CDROM"
     LogMsg "umount failed: ${sts}"
     LogMsg "Aborting test."
-    UpdateTestState "TestFailed"
+    SetTestStateFailed
     exit 1
 else
     LogMsg  "CDROM unmounted successfully"
 fi
 
 UpdateSummary "CDROM mount, read and remove operations returned no errors."
+
+#
+# Check without multiple "medium not present" in dmesg log
+# Refer to https://lkml.org/lkml/2016/5/23/332
+#
+logNum=`dmesg | grep -i "Medium not present" | wc -l`
+if [ $logNum -gt 1 ];then
+    LogMsg  "Multiple medium not present log show in the dmesg"
```

Comment:
Please also display this in the lisa ica.log test results summary, should be enough with an UpdateSummary.
If possible I would suggest this text instead for clarify:
"Error: Multiple "medium not present" messages found in dmesg"